cmake/ChooseCudaCC.cmake

```diff
@@ -40,14 +40,14 @@ function(ChooseCudaCC RESULT_NAME MINCC MINCUDAVERSION)
   #
   set(CUDA_MIN_CC 20)
   set(CUDA_MAX_CC 75)
-  if(CUDA_VERSION_MAJOR GREATER_EQUAL 10)
```

Comment:
if u want you'd better off forking my branch dev/cleaning where I upgraded cmake to 3.12 because otherwise it does not handle well cuda > 10

Reply:
nice, I'm trying to rebase in that case

Reply:
I've rebased it, but I reused the PR, so it still wants to merge into develop. I guess that's OK since dev/cleaning is going to be merged soon, right?

Reply:
I changed the destination base to dev/cleaning for the time being so we can see just the related diffs